starfish/test/full_pipelines/api/test_allen_smFISH.py

```diff
@@ -4,10 +4,11 @@
 import starfish.data
 from starfish import FieldOfView
 from starfish.image import Filter
-from starfish.spots import DetectSpots
+from starfish.spots import FindSpots
+from starfish.types import TraceBuildingStrategies
 
 
-@pytest.mark.skip('issues with checksums prevent this data from working properly')
+@pytest.mark.skip('This test runs but takes forever')
```

Comment:
hahahahaha

For what it's worth, there are a few allen smfish FOVs that have relatively few spots and run relatively quickly (2-3 minutes?)

Reply:
Do you know which ones?

Reply:
fov_004 has very few spots.